.devcontainer/devcontainer.json

```
@@ -2,7 +2,7 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.155.1/containers/alpine
 {
 	"name": "Alpine",
-	"image": "mcr.microsoft.com/vscode/devcontainers/base:0-alpine-3.12",
+	"image": "mcr.microsoft.com/vscode/devcontainers/base:0-alpine-3.20",
```

The recommended source for devcontainers is no longer the VS Code ones (see https://github.com/microsoft/vscode-dev-containers). The current source is at https://github.com/devcontainers/images. I'd go for the base alpine one (see https://github.com/devcontainers/images/tree/main/src/base-alpine).

For some reason the 3.21 seems not available though, so I'd just go for the 3.20.

```suggestion
	"image": "mcr.microsoft.com/devcontainers/base:alpine-3.20",
```